book/scheduling.md

```diff
@@ -393,15 +387,16 @@ Chapter 10 is not very useful and a huge performance footgun. Some
 browsers are now moving to deprecate synchronous `XMLHttpRequest`.]
 Python's `Thread` class lets us do better:
 
-    threading.Thread(target=callback).start()
+``` {.python .example}
+threading.Thread(target=callback).start()
+```
     
-This code creates a new thread and then immediately returns. The
-`callback` then runs in parallel, on the new thread, while the initial
-thread continues to execute later code.
+This code creates a new thread and then continues to execute. The
+`callback` then runs in parallel, on the new thread.
 
 We'll implement asynchronous `XMLHttpRequest` calls using threads.
-Specifically, we'll have the browser start a thread, do the request
-and parse the response on that thread, and then schedule a `Task` to
```

Comment:
This was hard to parse.

Reply:
Not sure I agree, the old text seems fine to me and new text is not critical to fix either

Reply:
Not critical, can revert, no strong feelings.

Reply:
Let's revert